rz-svd/meson.build

```diff
@@ -1,12 +1,31 @@
-project('rz-svd', 'c')
+project('rz-svd', 'c',
+  license: 'LGPL3',
+  meson_version: '>=0.55.0',
+  default_options: [
+    'buildtype=debugoptimized',
+    'b_vscrt=from_buildtype',
+  ]
+)
 
+# handle the yxml dependency
 plugins_dir = join_paths(get_option('prefix'), 'share', 'rizin', 'plugins')
+check_meson_subproject_py = files('sys/check_meson_subproject.py')
```

Comment:
ignore this... I think we don't need such level of user-friendliness yet for these. Just remove the whole check_meson_subproject stuff for now.